src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -325,7 +325,7 @@ def acr_build_task_run(cmd,
     logger.warning("Waiting for build agent...")
 
     if no_logs:
-        return get_build_with_polling(client, build_id, registry_name, resource_group_name)
+        return get_run_with_polling(client, build_id, registry_name, resource_group_name)
```

Comment:
run_id or id?

Reply:
@mnltejaswini I'll make a big item for this, thanks. https://github.com/AzureCR/azure-cli/issues/243